python/custreamz/custreamz/kafka.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+import confluent_kafka as ck
+from cudf_kafka._lib.kafka import KafkaDatasource
+
+import cudf
+
+from custreamz.utils import docutils
+
+
+# Base class for anything class that needs to interact with Apache Kafka
+class CudfKafkaClient(object):
```

Comment:
Don't need to subclass `object` as of Python 3:
```suggestion
class CudfKafkaClient:
```